crates/rpc/src/handlers/mod.rs

```diff
@@ -22,3 +27,26 @@ impl<T: Serialize> Data<T> {
         json(&json!(Self { data }))
     }
 }
+
+/// A generic data struct that can be used to wrap any data type
+/// used for json rpc responses
+///
+/// # Example
+/// {
+///  "data": json!(T)
+/// }
```

Comment:
```suggestion
/// # Example
/// {
///  "data": json!(T)
/// }
```

this example is wrong